build/release.mk

```diff
@@ -103,11 +103,23 @@ else
 	cp $(GOBIN)/darwin_amd64/mattermost $(DIST_PATH)/bin # from cross-compiled bin dir
 	cp $(GOBIN)/darwin_amd64/platform $(DIST_PATH)/bin # from cross-compiled bin dir
 endif
+	@# Prepackage plugins
+	@for plugin_package in $(PLUGIN_PACKAGES) ; do \
+		ARCH="osx-amd64"; \
+		cp tmpprepackaged/$$plugin_package-$$ARCH.tar.gz $(DIST_PATH)/prepackaged_plugins; \
+		cp tmpprepackaged/$$plugin_package-$$ARCH.tar.gz.sig $(DIST_PATH)/prepackaged_plugins; \
+		gpg --verify $(DIST_PATH)/prepackaged_plugins/$$plugin_package-$$ARCH.tar.gz.sig $(DIST_PATH)/prepackaged_plugins/$$plugin_package-$$ARCH.tar.gz; \
+		if [ $$? -ne 0 ]; then \
+			echo "Failed to verify" $$plugin_package-$$ARCH.tar.gz"|"$$plugin_package-$$ARCH.tar.gz.sig; \
+			exit 1; \
+		fi; \
```

Comment:
Would it make sense to add a test that pokes open the bundle and verifies the plugin binary exists for the given `$ARCH`?

Reply:
ah yes good call, i did it when i was testing my changes locally. Definitely be good to bake it into our build system.

I did it through grepping, couldn't figure out a better way for this.